feat(docs): add doc on choosing analytics systems (#35002)
gatsbyjs/gatsby: docs/docs/conceptual/choosing-analytics-systems.md

---
title: Choosing Analytics Systems
---

Website teams looking to collect insights on user behavior have a wide variety of options.

**Basic Analytics**, which includes [Google Analytics](/plugins/gatsby-plugin-google-analytics/?=google%20analytics), focus on metrics like page views, sessions, bounce rate, entrances and exits, measured on a per-URL basis and tracked over time. They include UTM parameter tracking, in order to tie visitors to specific paid advertising or email campaigns.

Basic analytics tools are often used alongside **Tag Managers** ([Google Tag Manager](/plugins/gatsby-plugin-google-tagmanager/?=google%20tag%20manager)), which can let marketers configure which pages load tracking scripts, or define events when certain elements are clicked.

Google Analytics and Google Tag Manager are the most popular services used on Gatsby sites. But over the last few years, many specialized analytics services have emerged to give deeper insight into visitor behavior.

Gatsby teams have primarily reached for three categories of tools:

> **Customer Data Platforms** ([Segment](/plugins/gatsby-plugin-segment-js/?=segment), [Rudderstack](/plugins/gatsby-plugin-rudderstack/?=rudderstack)) are a tool to help teams manage and orchestrate multiple analytics systems. This includes centralizing data on visitors.

Reviewer: Grammar suggestion: 
> **Customer Data Platforms** are tools that help teams...

or...

> **Customer Data Platforms** help teams...

Author: Thx, fixed! Back over to you.

> **Customer Data Platforms** ([Segment](/plugins/gatsby-plugin-segment-js/?=segment), [Rudderstack](/plugins/gatsby-plugin-rudderstack/?=rudderstack)) are a tool to help teams manage and orchestrate multiple analytics systems. This includes centralizing data on visitors.

Author: ```suggestion
**Customer Data Platforms** ([Segment](/plugins/gatsby-plugin-segment-js/?=segment), [Rudderstack](/plugins/gatsby-plugin-rudderstack/?=rudderstack)) are tools that help teams manage and orchestrate multiple analytics systems. This includes centralizing data on visitors.
```
